scripts/mpinfoin.pl

```diff
@@ -158,7 +158,11 @@
 my $personinfoupdate = $dbh->prepare('update personinfo set data_value=? where person_id=? and data_key=?');
 my $consinfoadd      = $dbh->prepare("insert into consinfo (constituency, data_key, data_value) values (?, ?, ?) on duplicate key update data_value=?");
 
+my $start_transaction = $dbh->prepare("START TRANSACTION");
```

Comment:
This presumably works, but an easier way would be to call `$dbh->{AutoCommit} = 0;`, then no need to start any transactions, and can call `$dbh->commit` to commit, then set it back to 1 afterwards.
Does putting these in a transaction speed them up? Bit surprising to me.